src/main/java/gov/usgs/owi/nldi/springinit/SpringConfig.java

```diff
@@ -29,6 +40,39 @@ public InternalResourceViewResolver setupViewResolver() {
 		return resolver;
 	}
 
+	@Override
+	public void configureContentNegotiation(ContentNegotiationConfigurer configurer) {
+		configurer.strategies(Arrays.asList(new ContentNegotiationStrategy() {
```

Comment:
I presume the IOW normal pattern did not work? https://github.com/NWQMC/WQP-WQX-Services/blob/fdd94bd2c4d0d12dfb5b9dc653a9c34359bd95e2/src/main/java/gov/usgs/wma/wqp/springinit/SpringConfig.java#L67

Reply:
Yes, I tried that pattern and could not get the correct behavior all the time.